sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/RoslynDataImporter.java

```diff
@@ -55,4 +60,11 @@ public void importRoslynReport(List<Path> reportPaths, final SensorContext conte
     }
     return repositoryKeyByRoslynRuleKey;
   }
+
+  private static String pluralize(String s, int count) {
```

Comment:
Just out of curiosity don't you have any built-in method for this?

Reply:
No